src/components/profile/verificationModals/confirmVerificationModal.tsx

```diff
@@ -75,7 +75,11 @@ const ConfirmVerificationModal = (props: ConfirmVerificationModalProps) => {
                 name="verificationCode"
                 id="verificationCode"
                 className="fs-2 text-center"
-                style={{ letterSpacing: "0.5em", paddingLeft: "0.5em" }}
+                style={{
+                  letterSpacing: "0.5em",
+                  paddingLeft: "0.5em",
+                  width: "8em",
```

Comment:
```suggestion
                  width: "7em",
```

To make it the same size as for the 2FA code.

Reply:
The 8em was a fix for Edge, since 7em was too narrow for the IVA verification code, so I made the TOTP one also 8em instead